test/Microsoft.ML.Tests/Transformers/FeatureSelectionTests.cs

```diff
@@ -182,9 +182,9 @@ public void MutualInformationSelectionWorkout()
 
             var est = ML.Transforms.FeatureSelection.SelectFeaturesBasedOnMutualInformation("FeatureSelect", "VectorFloat", slotsInOutput: 1, labelColumnName: "Label")
                 .Append(ML.Transforms.FeatureSelection.SelectFeaturesBasedOnMutualInformation(labelColumnName: "Label", slotsInOutput: 2, numberOfBins: 100,
-                    columns: new ColumnOptions[] {
-                        ("out1", "VectorFloat"),
-                        ("out2", "VectorDouble")
+                    columns: new[] {
+                        new InputOutputColumnPair("out1", "VectorFloat"),
```

Comment:
>InputOutputColumnPair [](start = 28, length = 21)

why do we have to use the more verbose initializer here?  Ideally we should use the old syntax is possible as it is a lot more compact.

Reply:
Because the other one uses tuples. We have decided not to have tuples in the public surface any longer.

---
In reply to: [268729946](https://github.com/dotnet/machinelearning/pull/3066#discussion_r268729946) [](ancestors = 268729946)